doc: Added the notes of the theory pills

alejandromd opened the pull request:
# Description

I have created a new section in documentation where I have added all the notes collected from the theory pills to date.

# Checklist:
- [x] I have performed a self-review of my code running `npm run build` and `npm run serve` after pushing the code.
- [x] I have checked that there is no fragile information published unintentionally.

# Screenshot:

![pr](https://github.com/ISPP-07/htld-doc/assets/72470942/655ec4a1-ef53-4327-bae4-217d95bc5b64)

AntonioRodriguezRuiz commented:
De momento he revisado la primera pildora teorica. En general esta muy bien pero hay un par de cosas que me gustaria que concretaras un poquito mas:
- El CI se debe hacer en todas las ramas, antes y despues de mergear una PR
- El apartado "Accepted" solo es util cuando se tiene un cliente real
- Si no recuerdo mal de como media BlueJay el cumplimiento de estándares, se necesita un comentario por persona por semana en las PR, solo aprobarla no funcionaba
- El MVP debe estar **Terminado** antes del lanzamiento, no **planificado**

AntonioRodriguezRuiz commented:
The second part is pretty good, nothing to add there.

alejandromd commented:
> So far I've checked the first theory pill. In general its pretty good but there are some thing I'd like you to improve on:
> 
> * El CI se debe hacer en todas las ramas, antes y despues de mergear una PR
> * El apartado "Accepted" solo es util cuando se tiene un cliente real
> * Si no recuerdo mal de como media BlueJay el cumplimiento de estándares, se necesita un comentario por persona por semana en las PR, solo aprobarla no funcionaba
> * El MVP debe estar **Terminado** antes del lanzamiento, no **planificado**

- I have corrected the metrics according to the document "ISPP2324-TeamPracticesDescriptionv-1.0.pdf". 
![image](https://github.com/ISPP-07/htld-doc/assets/72470942/90941c11-a0cb-4de5-a33a-3be12fdc00ef)

- I have not changed "Accepted" since they are notes taken from the theoretical pills in order to take the test, so I consider leaving it.

- I have changed the verb to "Finished"

alejandromd commented:
> So far I've checked the first theory pill. In general its pretty good but there are some thing I'd like you to improve on:
> 
> * El CI se debe hacer en todas las ramas, antes y despues de mergear una PR
> * El apartado "Accepted" solo es util cuando se tiene un cliente real
> * Si no recuerdo mal de como media BlueJay el cumplimiento de estándares, se necesita un comentario por persona por semana en las PR, solo aprobarla no funcionaba
> * El MVP debe estar **Terminado** antes del lanzamiento, no **planificado**

- I have corrected the metrics according to the document "ISPP2324-TeamPracticesDescriptionv-1.0.pdf". 
![image](https://github.com/ISPP-07/htld-doc/assets/72470942/9f338ccb-fd29-46bc-ad00-7b2655e80954)

- I have not changed "Accepted" since they are notes taken from the theoretical pills in order to take the test, so I consider leaving it.

- I have changed the verb to "Finished"
